src/js/letters/actions/letters.js

```diff
@@ -118,8 +104,11 @@ export function getMailingAddress() {
           data: responseCopy
         });
       },
-      // catch errors in fetch or success handler
```

Comment:
I guess the success handler doesn't throw any errors, does it? Good catch (pun unfortunately intended).